client/src/components/banks/BankDetails/BankDetails.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useContext } from "react";
+import { Button } from "antd";
+
+import { Web3Context } from "../../../contexts/RootContexts";
+import { truncateAddr } from "../../../utils/helpers";
+import EtherscanLink from "../../shared/EtherscanLink/EthercanLink";
+import CreateVault from "../../vaults/CreateVault/CreateVault";
+import Web3SignIn from "../../account/Web3SignIn";
+
+import "./BankDetails.scss";
+
+const BankDetails = ({ address, bank }) => {
+  const [web3] = useContext(Web3Context);
+  const [creatingVault, setCreatingVault] = useState(false);
+  const data = bank.data;
+
+  return (
+    <>
+      <div className="BankDetails">
```

Comment:
The two tokens and their prices should appear in the bank details rather than at the top of the page. Banks will use many different tokens, so having TRB DAI prices at the top outside the bank details tiles won't make sense once another token is added (e.g. PNK, RARI, etc.)
![Screen Shot 2020-08-28 at 9 46 16 AM](https://user-images.githubusercontent.com/5077880/91569225-be9f9000-e913-11ea-96be-f6587b39c40b.png)